easybuild/easyconfigs/p/pytest/pytest-7.4.2-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,96 @@
+easyblock = 'PythonBundle'
+
+name = 'pytest'
+version = '7.4.2'
+
+homepage = 'https://docs.pytest.org/en/latest/'
+description = """The pytest framework makes it easy to write small,
+readable tests, and can scale to support complex functional testing for
+applications and libraries."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+builddependencies = [
+    ('binutils', '2.40'),
+]
+
+dependencies = [
+    ('Python', '3.11.3'),
```

Comment:
@ItIsI-Orient `Python-bundle-PyPI` should also be a dependency here, and `exts_list` should be cleaned up accordingly (remove `packaging` + `tomli` from this easyconfig, maybe more)